api/tests/Api/CampCollaborations/CreateCampCollaborationTest.php

```diff
@@ -11,9 +11,6 @@
  * @internal
  */
 class CreateCampCollaborationTest extends ECampApiTestCase {
-    // TODO input filter tests
-    // TODO validation tests
```

Comment:
Könnte man diese Kommentare jetzt noch in anderen Testdateien löschen, z.B. bei CreateActivityTest? Oder fehlt dort noch etwas?

Reply:
Jup, das habe ich vergessen, sorry. Hole ich sobald wie möglich nach.

Reply:
done with fixup commits, will rebase them later